01/highest-rated.md

```diff
@@ -42,19 +103,98 @@ Consider the following for inspiration:
 ##### !end-hint
 ##### !explanation 
 
-Example clarifying questions:
+Example clarifying questions and reasoning:
 
 1. What should the function return if there are multiple restaurants tied for the highest rating?
-1. How should the function handle restaurant dictionaries with missing data (name or rating)?
-1. How should the function handle restaurant dictionaries with keys in addition to `"name"` and `"rating"`?
-1. How should the function handle non-numerical values for `"rating"`?
-1. How should the function handle input that is a single dictionary (not a list)?
+    - There could be multiple restaurants with the same rating, I want to know if it is important whether we return the first found vs. last found
+    - I will return the first restaurant found with the highest rating
+2. Can the `rating` value be a float, or will it only ever be an integer?
+    - I want to know if we need to consider fractional numbers at all when designing our implementation
+    - I will assume the rating will only be integers as in the examples
+3. Can the list contain duplicate restaurant entries (same name and rating)?
+    - I want to know if fully duplicate restaurants should be filtered out or considered like any other restaurant
+    - I will assume that we are given a list of unique restaurants with no duplicates
+4. How should the function handle non-numerical values for `rating`?
+    - We cannot compare two restaurants if the rating is missing, I want to know if we should try to convert the value to a number, skip the restaurant, or raise an error
+    - Since this is around error handling, I will assume error handling has been managed outside the function and that the data received by the function has a valid format
  
 ##### !end-explanation
 ### !end-challenge
 <!-- prettier-ignore-end -->
 
-<!-- Question 2 -->
+### !challenge
+* type: paragraph
+* id: f9ad5047-e187-4213-9a8a-fae368cf813b
+* title: Review Clarifying Questions
+* topics: pse
+##### !question
+
+While we build our skills in breaking down a problem and choosing clarifying questions, let’s use an external tool like ChatGPT to review the questions we wrote above. Our goals are to: 
+- ensure we are asking questions that will tell us new information about the problem space
+- check our understanding of the information we expect to get from those questions
+- uncover other questions that could be useful to ask
+- understand why those other questions could be helpful
+
+<br>
+
+For this question we will:
+1. Build a prompt using [the template linked here](../shared-resources/questions-review-prompt-template.md)
```

Comment:
When I click this link it opens the file as unformatted markdown. Is that what we're expecting?

![image](https://github.com/user-attachments/assets/580543ed-26fe-4b10-90ae-1ebd26329315)

If not, I haven't figured out why the file isn't opening up as formatted markdown.

Reply:
Hmm, I was hoping it would render the formatting, but the only actual markdown is the title "Template for ...", so I'm not sure it would be an issue for students. We could always link to the file in GitHub rather than linking to the file in Learn if that's a concern.